.github/workflows/publish_build.yml

```
@@ -0,0 +1,84 @@
+        with:
+          ref: dev
+          fetch-depth: 0 # otherwise, there would be errors pushing refs to the destination repository.
+      - name: Setup Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: 3.10
+      - name: Install Build Tools
+        run: |
+          python -m pip install build wheel
+      - name: Remove alpha (declare stable)
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2034:warning:1:1: VER appears unused. Verify use (or export if used externally)

(shellcheck)